boards/nrf52840-dongle/Makefile.include

```diff
@@ -0,0 +1,38 @@
+# This board uses the vendor's serial bootloader
+
+PROGRAMMER ?= nrfutil
+
+# We're not using the softdevice actually (nor use JLink as in
+# nrf52/Makefile.include), but the linker script should be a good starting
+# point for using the nRF bootloader and not breaking the soft device.
+
+LINKER_SCRIPT ?= $(BOARDSDIR)/nrf52840-dongle/ldscripts/application.ld
+
+# This is under active exploration, and should probably go into its one
+# makefiles/tools file
+
+ifeq (nrfutil,$(PROGRAMMER))
+
+FLASHFILE = $(HEXFILE).zip
+FLASHER = nrfutil
+FFLAGS = dfu usb-serial --port=${PORT} --package=$(HEXFILE).zip
+
+RESET ?= false
```

Comment:
Not 100% sure, but I think using the `false` command for flashing here could lead to problems running tests. @fjmolinas do you know about that?

Reply:
Is it set to false because reset is not possible? I would rather leave the definition empty in that case:

```
RESET ?= 
```

Reply:
If I leave it empty, a `make reset` will succeed but not have reset the board.

Unless that target is clearly marked as best-effort I'm more worried about mechanisms that depend on a reset to happen failing silently, and not so much about mechanisms that don't really depend on a reset but kind of think it'd be good to have the board reset now receiving a clear indication that this can not work on the setup. (The latter can still `make reset || true` or similar, the other way round it's a lot harder).

Reply:
> If I leave it empty, a `make reset` will succeed but not have reset the board.

I believe that is how all our boards that do not support reset behave.

Reply:
Alright, then -- I've removed the `RESET ?=` line, which apparently makes reset a no-op.